helpers/class.ResultTransmitter.php

```diff
@@ -182,6 +185,11 @@ private static function transformValue($value) {
             if ($value instanceof QtiFile) {
                 return taoQtiCommon_helpers_Utils::qtiFileToString($value);
             }
+            if ($value instanceof RecordContainer) {
+                $marshaller = new Marshaller;
```

Comment:
PSR12:  Parentheses must be used when instantiating a new class

```suggestion
                $marshaller = new Marshaller();
```